scripts/evaluate_model_entropy.py

```diff
@@ -0,0 +1,78 @@
+"""Script to evaluate learned predicate classifiers on held-out test cases."""
+
+from typing import Sequence
+
+import numpy as np
+
+from predicators.scripts.evaluate_predicate_classifiers import main
```

Comment:
Can we rename main to something more descriptive if it's gonna be used in multiple files?